### ISSUE TITLE
Axe dev tools pro recurring audit-main

### ISSUE THREAD

**Issue by amycole501** [GSA/px-benefit-finder#58]
Once per month an a11y team member will perform a manual audit of a section of the benefits finder website using axe dev tools. Both [random and focused](https://library.bixal.com/books/testing-remediating/page/focused-and-random-accessibility-testing) can be used.

Options for audits include:
Form validation
Form header/footer
QA testing for radio buttons

Testing on mobile
Testing in Edge
Testing in Firefox

The issues from the previous month's report can be submitted to the dev team for tickets.

The definition of done includes generating a report for a specific field, WCAG success criteria or interaction and sharing findings in the [Google folder](https://drive.google.com/drive/folders/1-_iiBmMpgq21fghY86nEK8aNxB9ZZama?usp=sharing).

**Comment by scottqueen-bixal** [comment]
Added this ticket -> https://github.com/orgs/GSA/projects/48/views/1?pane=issue&itemId=47208573

to include AXE scanning in our pipeline as well - we can coordinate with @nehemiah-abuga to see that we are configured correctly in line with the audit work the a11y team is implementing.

**Comment by fongcindy** [comment]
@angelina-smith Any updates on this? Is this a recurring item?

**Comment by angelina-smith** [comment]
If @nehemiah-abuga  is including axe into the QA process, I'm not sure we need this ticket anymore? If we are not currently doing this, then it would be a recurring ticket!

**Comment by scottqueen-bixal** [comment]
> If @nehemiah-abuga is including axe into the QA process, I'm not sure we need this ticket anymore? If we are not currently doing this, then it would be a recurring ticket!

Agree @angelina-smith - let's have @nehemiah-abuga revisit this test suit and make sure we are set up with the following considerations 

- covering positive and negative scenarios to confirm functionality of testing 
- testing all scenarios which render html states
- fails on error
- outputs errors
- etc.

**Comment by nehemiah-abuga** [comment]
I have reviewed the axe tests. Everything looks good.
